src/main/kotlin/coverosR3z/server/utility/RoutingUtilities.kt

```diff
@@ -48,6 +48,7 @@ class RoutingUtilities {
                 Pair(Verb.POST, LoggingAPI.path) -> LoggingAPI.handlePost(sd)
                 Pair(Verb.POST, ViewTimeAPI.path) -> ViewTimeAPI.handlePost(sd)
                 Pair(Verb.POST, SubmitTimeAPI.path) -> SubmitTimeAPI.handlePost(sd)
+                Pair(Verb.POST, UnsubmitTimeAPI.path) -> UnsubmitTimeAPI.handlePost(sd)
```

Comment:
I'm still brewing on the concept of each API having the ability to add itself to the routing table, a dynamic list maybe.  But on the other hand, having it this way means that there is a "table of contents" of the endpoints in one spot, and the stacktraces look nicer too.